<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrograPop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="nombre_usuario">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ingrese_nombre">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese su nombre:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_ingreso">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>280</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingresar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Baskerville</family>
        <pointsize>64</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>PrograPop</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_puntaje">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Puntaje: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_usuario">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>PT Mono</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_puntajes">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Puntajes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="nombre_sala_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>48</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>271</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="opcion1">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="opcion2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="opcion3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="opcion4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="boton_volver">
      <property name="geometry">
       <rect>
        <x>652</x>
        <y>40</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
     <widget class="QLabel" name="pregunta">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>¿Qué canción esta sonando?</string>
      </property>
     </widget>
     <widget class="QLabel" name="timer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>445</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Menlo</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>timer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_puntaje_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Puntaje: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="correcto">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>150</y>
        <width>811</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Esperando canción
siguiente...</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>80</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Usuarios en la sala</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>48</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Puntajes Usuarios</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>801</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Usuario</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Puntaje</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Mejor Sala</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Peor Sala</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>-3</x>
        <y>130</y>
        <width>801</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>21</width>
        <height>441</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>21</width>
        <height>441</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>21</width>
        <height>441</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="volver_puntajes">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
